[2025-06-03T07:08:43.565+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prepare_churn_dataset.create_table scheduled__2025-06-03T00:00:00+00:00 [queued]>
[2025-06-03T07:08:43.573+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prepare_churn_dataset.create_table scheduled__2025-06-03T00:00:00+00:00 [queued]>
[2025-06-03T07:08:43.574+0000] {taskinstance.py:1361} INFO - Starting attempt 6 of 6
[2025-06-03T07:08:43.586+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2025-06-03 00:00:00+00:00
[2025-06-03T07:08:43.593+0000] {standard_task_runner.py:57} INFO - Started process 458 to run task
[2025-06-03T07:08:43.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'prepare_churn_dataset', 'create_table', 'scheduled__2025-06-03T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/churn.py', '--cfg-path', '/tmp/tmpdzh6g8hv']
[2025-06-03T07:08:43.598+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask create_table
[2025-06-03T07:08:43.652+0000] {task_command.py:416} INFO - Running <TaskInstance: prepare_churn_dataset.create_table scheduled__2025-06-03T00:00:00+00:00 [running]> on host 2ad5921a312f
[2025-06-03T07:08:43.729+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prepare_churn_dataset' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-03T00:00:00+00:00'
[2025-06-03T07:08:43.733+0000] {logging_mixin.py:154} INFO - hook:  <***.providers.postgres.hooks.postgres.PostgresHook object at 0x7f91ca60c370>
[2025-06-03T07:08:43.740+0000] {base.py:73} INFO - Using connection ID 'destination_db' for task execution.
[2025-06-03T07:08:43.742+0000] {logging_mixin.py:154} INFO - db_conn:  Engine(postgresql://mle_20250507_60d03b0a2f_freetrack:***@rc1b-uh7kdmcx67eomesf.mdb.yandexcloud.net:6432/playground_mle_20250507_60d03b0a2f)
[2025-06-03T07:08:43.744+0000] {logging_mixin.py:154} INFO - users_churn:  users_churn
[2025-06-03T07:08:43.744+0000] {logging_mixin.py:154} INFO - users_churn:  users_churn.customer_id
[2025-06-03T07:08:43.772+0000] {logging_mixin.py:154} INFO - create table!
[2025-06-03T07:08:43.815+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-03T07:08:43.827+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_churn_dataset, task_id=create_table, execution_date=20250603T000000, start_date=20250603T070843, end_date=20250603T070843
[2025-06-03T07:08:43.889+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-03T07:08:43.927+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
